README.md

```diff
@@ -69,43 +80,53 @@ As fallback, members can also respond with a Multicast/UDP message.
   "deviceModel": "Windows",
   "deviceType": "desktop",
   "fingerprint": "random string",
-  "announcement": false
+  "announce": false
 }
 ```
 
 The `fingerprint` is only used to avoid self-discovering.
 
-A response is only triggered when `announcement` is `true`.
+A response is only triggered when `announce` is `true`.
 
-### 2.2 HTTP (Legacy Mode)
+### 3.2 HTTP (Legacy Mode)
 
 This method should be used when multicast was unsuccessful.
 
 Devices are discovered by sending this request to all local IP addresses.
 
-`GET /api/localsend/v1/info?fingerprint=abc`
+`POST /api/localsend/v2/register`
+
+Request
+
+```json5
+{
+  "alias": "Secret Banana",
+  "deviceModel": "Windows",
+  "deviceType": "desktop",
+  "fingerprint": "random string" // ignored in HTTPS mode
+}
+```
 
 Response
 
 ```json5
 {
   "alias": "Nice Orange",
-  "deviceModel": "Samsung", // nullable
-  "deviceType": "mobile" // mobile | desktop | web
+  "deviceModel": "Samsung",
+  "deviceType": "mobile",
+  "fingerprint": "random string" // ignored in HTTPS mode
 }
 ```
 
-## 3. File transfer (HTTP)
+## 4. File transfer (HTTP)
 
-### 3.1 Send Request (Metadata only)
+### 4.1 Send Request (Metadata only)
 
 Sends only the metadata to the receiver.
 
 The receiver will decide if this request gets accepted, partially accepted or rejected.
 
-Be aware that only one active session is allowed, i.e. the server will reject all requests when another session is ongoing.
-
-`POST /api/localsend/v1/send-request`
+`POST /api/localsend/v2/send-request`
```

Comment:
i think, rename this api as `prepare-request` better

Reply:
There will be `/receive-request` and `/receive` when https://github.com/localsend/localsend/issues/198 gets implemented.
The suffix `-request` indicates that it may be rejected. Let me know if your ideas!

Reply:
I changed the routes to `prepare-upload`, `upload`, `prepare-download`, `download`.